docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,142 @@
+
+
+Definitions
+-----------
+
+
+Source project
+~~~~~~~~~~~~~~
+
+For a given documentation project to be assessed, the *source project* comes into question. This is defined as:
+
+* The origin project from which the documentation project's repository was forked (if forked).
+* The origin project from which the documentation project was copied (if not forked but evidently copied).
+* The original project which the documentation project refers as the subject of documentation.
+* Any other project that is understood by the core team to be a the *source project*.
+
+
+Unofficial projects
+~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unofficial* when it is not linked to or mentioned in *any* of the following places:
+
+* Websites and domains associated with the project
+* The project's primary repository -- README files, repository description, or source code
+
+
+Unmaintained projects
+~~~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unmaintained* when ANY of the following are met:
+
+* The configured version control repository is unreadable. This can happen if the repository is deleted, credentials are broken or the Git host is permanently unresponsive.
+* The project is only serving content from releases and commits 6 months or older than its source repository.
+* All builds have failed for more than 6 months.
+
+
+Reachability
+~~~~~~~~~~~~
+
+In every case where contacting the user is necessary, the core team will reach out using one of the following means of contact:
+
+* E-mail address on file in the user's profile
+* E-mail address on the GitHub user account where the project is pointed
+
+The core team will stop trying to reach the user after 3 weeks and the user will be considered *unreachable*.
+
+
+Implementation
+--------------
+
+
+Delisting of an abandoned project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Projects that are determined to be unmaintained_ and unofficial_ will have a ``robots.txt`` file added that removes them from all search results:
+
+.. code-block:: text
+
+  # robots.txt
+  User-agent: *
+  # This project is delisted according to the Policy for Unofficial and Unmaintanied Projects
+  # https://docs.readthedocs.io/en/stable/unofficial-projects.html
+  Disallow: /
+
+
+Projects will be delisted if they meet *all* of the following criteria:
+
+* The person who submits the report of the unmaintained_ and unofficial_ project also demonstrates failed attempts to contact the existing owners.
+* The project has been determined unmaintained_ and unofficial_ by the rules described above.
+* The project owners is notified by the Read the Docs team.
+* The core team does not have any additional reservations.
+* If the project owners are unreachable_ by the Read the Docs team, no further action is taken.
+* If the any of the project owners respond, their response will be taken into account by the core team and it may be listed again if the criteria are met.
```

Comment:
```suggestion
* If any of the project owners respond, their response will be taken into account by the core team and it may be listed again if the criteria are met.
```